gcc/rust/hir/tree/rust-hir-expr.h

```diff
@@ -3387,6 +3387,8 @@ class WhileLetLoopExpr : public BaseLoopExpr
 
   void accept_vis (HIRVisitor &vis) override;
 
+  std::unique_ptr<Expr> &get_cond () { return condition; }
```

Comment:
is this used?

Reply:
Yes, in WhileLetLoopExpr visit, I need to visit the cond.

Reply:
Sorry I missed that :)